app/controllers/users.js

```diff
@@ -0,0 +1,14 @@
+const { inspect } = require('util');
+
+const logger = require('../logger');
+const { createUser, encryptPassword } = require('../services/users');
+const errors = require('../errors');
+
+exports.signUp = ({ body }, res, next) =>
+  encryptPassword(body.password)
```

Comment:
Remember always log the activity, like `Encrypting the password`, `Creating user`, `User created successfully`, etc